academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md

```diff
@@ -35,15 +47,17 @@ A release may not be completed when you see a new folder in your storage account
 
 ## MAKES hosting resource/deployment region mismatch
 
-MAKES can only be deployed to the same region as the MAKES hosting image. If you see an error message containing "...The Image '/subscriptions/XXXXXXX-XXXX-XXX-XXXX-XXXXXXX/resourceGroups/xxxxmakeshostres/providers/Microsoft.Compute/images/xxxxmakeshostres' cannot be found in 'xxxx' region..." Make sure to create MAKES deployments and hosting images in the same region.
+When running **DeployHost** command, If you encounter an error messages containing "...The Image '/subscriptions/XXXXXXX-XXXX-XXX-XXXX-XXXXXXX/resourceGroups/xxxxmakeshostres/providers/Microsoft.Compute/images/xxxxmakeshostres' cannot be found in 'xxxx' region...", make sure to create MAKES deployments and hosting images in the same region.
```

Comment:
>make sure to create MAKES deployments and hosting images in the same region [](start = 266, length = 75)

"make sure that both your MAKES host image resources and deployments are in the same region"